cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -1366,18 +1380,23 @@ def _compute_visibility_state(
         all_staff_only = True
         all_unscheduled = True
         all_live = True
-        for child in child_info["children"]:
+        all_hide_from_toc = True
+        for child in child_info["children"]:  # pylint: disable=too-many-nested-blocks
```

Comment:
Note to reviewer: this is not the best way of resolving this warning, but this PR is complex enough as it is. So I'll be opening a new PR proposing a refactor change so this quality failure is not raised. anymore